examples/demo-cilium-egress/exec-sandbox/server.py

```
@@ -0,0 +1,57 @@
+        else:
+            self._send(404, {"error": "not found"})
+
+    def do_POST(self):
+        if self.path != "/execute":
+            self._send(404, {"error": "not found"})
+            return
+        n = int(self.headers.get("Content-Length", 0) or 0)
+        raw = self.rfile.read(n) if n else b"{}"
+        try:
+            cmd = json.loads(raw or b"{}").get("command", "")
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)